contracts/bonding/BondingManager.sol

```diff
@@ -82,7 +82,7 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
     // and instead calculates the total bonded value separately
     uint256 private totalBondedDEPRECATED;
 
-    // Candidate and reserve transcoders
+    // Candidate active transcoder set for 'currentRound + 1'
```

Comment:
Don't need this change since the comment on the `streamflow` branch is updated to `DEPRECATED - DO NOT USE` since we deprecated `transcoderPool`.

Reply:
fixed in b79bb67